internal/cli/root.go

```diff
@@ -86,23 +64,10 @@ func NewClient(consumerToken, apiURL, Version string) *Client {
 	}
 }
 
-// This function provides backwards compatibility for the packngo
```

Comment:
This comment should stay since we haven't settled on or introduced a new debug variable